go.mod

```diff
@@ -1,33 +1,122 @@
 module github.com/osmosis-labs/osmosis/v3
 
-go 1.15
+go 1.17
```

Comment:
Why not just bump to 1.18 (including our CI plumbing) while we're at it?

Reply:
Absolutely